aev/src/main/java/com/fingerprintjs/android/aev/AevClientFactory.kt

```diff
@@ -50,11 +50,12 @@ object AevClientFactory {
             ossInstance,
             getApiInteractor(
                 endpointUrl,
-                getAppName(applicationContext),
+                getAppName(applicationContext).toString(),
```

Comment:
Why do we need `.toString()` here? Is there possible to cause NPE?

Reply:
That is because for the sake of safety `getAppName` has nullable return type now. This change is aimed to preserve our previous logic.